src/elements/content-sidebar/versions/VersionsItem.scss

```diff
@@ -21,6 +21,11 @@
         position: absolute;
         top: $bcs-VersionsItemButton-vertical;
         right: $bcs-VersionsItemButton-horizontal;
+
+        &:focus {
+            border: $bdl-btn-border-width solid $primary-color;
+            box-shadow: inset 0 0 0 1px fade-out($white, .2), 0 1px 2px fade-out($black, .9);
```

Comment:
Is this where you copied this from https://github.com/box/box-ui-elements/blob/4d71c6279313b24e9529c9f8b769348453a57226/src/styles/common/_buttons.scss#L179

Reply:
So I'm guessing that those values are supposed to be consistent i.e. if one changes the other should change as well? If that's the case I'd recommend creating a mixin for this and use @include in both places. If they don't have to be the same based on our overall design scheme then we can just leave it alone. Thoughts?

Reply:
The main problem was that this button has "a low visibility focus indicator." I think it doesn't need to be exactly the same as primary button one but I can move it to the mixin to keep consistency.